packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1927,6 +1928,12 @@ class Scaffold extends StatefulWidget {
   ///    Flutter.
   final String? restorationId;
 
+
+  /// Called when the status bar is tapped. (iOS only)
```

Comment:
This does not feel appropriate as iOS only. Other devices have a status bar no?

Reply:
Other devices have status bar, but only iOS supported that gesture. Then, how about rename this event to `oniOSStatusBarTapped`?

I  just wanted to make a name that would be unified with private method below

```
  // iOS FEATURES - status bar tap, back gesture

  // On iOS, tapping the status bar scrolls the app's primary scrollable to the
  // top. We implement this by looking up the primary scroll controller and
  // scrolling it to the top when tapped.
void _handleStatusBarTap() {
```

Reply:
Other devices have status bars though, right? Why limit this to iOS?